lib/Command/Scan.php

```diff
@@ -56,7 +56,7 @@ protected function configure() {
 
 	protected function execute(InputInterface $input, OutputInterface $output) {
 		$folderId = $input->getArgument('folder_id');
```

Comment:
```suggestion
		$folderId = (int) $input->getArgument('folder_id');
```

cast here, so that the variable is save to use for potential future usage?

Reply:
Make sense